### PR TITLE
Checkpages

### PR DESCRIPTION
**Pull request by hadobe**
**Fixes following issues|Adds following features:**
- Check Adobe Express pages for 404 errors.

**Resolves:** [MWPW-157784](https://jira.corp.adobe.com/browse/MWPW-157784)

**Steps to test the before vs. after and expectations:**
- npx run test/e2e -t "@smoke-test" "@regression"

**Pages to check for regression and performance:**
- https://<branch>--express--adobecom.hlx.page/express/?martech=off

### CODE REVIEW

File: test/e2e/urls.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  - "https://www.adobe.com/express/feature/video/animate/audio"
+  - "https://www.adobe.com/express/feature/video/add-caption"
+  - "https://www.adobe.com/express/feature"
+  - "https://www.adobe.com/express/feature/video/remove-background"
+  - "https://www.adobe.com/express/business/teams"
+  - "https://www.adobe.com/express/business"
+  - "https://www.adobe.com/express/nonprofits"
+  - "https://www.adobe.com/express/entitled"
+  - "https://www.adobe.com/express/business#sales-contact-form-1"
+  - "https://www.adobe.com/express/#open-jarvis-chat"
```
Comment (vhargrave) [inline test/e2e/urls.yml:62]: this is the same url as https://www.adobe.com/express/

File: test/e2e/profiles.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+default: -p prod -b chromium --headless false --logConsole
+
+prod: 
+  baseUrl: https://www.adobe.com/express/
+
+stage:
+  baseUrl: https://www.stage.adobe.com/express/
```
Comment (vhargrave) [inline test/e2e/profiles.yml:8]: can you even test on stage?

**Review by vhargrave — COMMENTED** [review]
Wow, interesting. 
I'm right now a little uncertain if we need these tests here though. Is there a reason not to just keep everything in nala?  - https://github.com/adobecom/nala/blob/main/tests/express/check-important-pages.test.js

**Comment by hadobe** [comment]
closing the PR as we will not be working on this for now.

**Pull request closed without merging**